src/util/GeneratorConverters.h

```diff
@@ -0,0 +1,39 @@
+#include <optional>
+
+#include "util/Generator.h"
+#include "util/Iterators.h"
+
+template <typename ValueType>
+cppcoro::generator<ValueType> fromInputRange(
+    ad_utility::InputRangeTypeErased<ValueType> inputRange) {
+  for (auto& value : inputRange) {
+    co_yield value;
+  }
+}
```

Comment:
This only is used in its own unit test, can it be removed?

Reply:
definitely yes, it was just there to migrate coroutines file by file with patching existing cppcoro